### ISSUE TITLE
Infer from usage codefix should work in JS 

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/TypeScript#20113]
**TypeScript Version:**  2.7.0-dev.20171117

**Code**

```js
function f(x) {
    return x + 1;
}
```

**Expected behavior:**

```js
/** @param {number} x */
function f(x) {
    return x + 1;
}
```

**Actual behavior:**

No fix.

**Comment by andy-ms** [comment]
Similarly, we could add jsdoc annotations in `fixAddMissingMember.ts` for JS like we do for TS.

**Comment by sandersn** [comment]
Fixed by #27610; I guess github doesn't use comments to auto-close bugs.
